be/src/common/configbase.cpp

```diff
@@ -308,6 +318,35 @@ bool init(const char* filename, bool fillconfmap) {
         SET_FIELD(it.second, std::vector<std::string>, fillconfmap);
     }
 
+    // if user hasn't set flush_thread_num_per_store, then we will set
+    // default value to the flush_thread_num_per_store based on the disk num and core num
+    if (!props.contain("flush_thread_num_per_store")) {
+        size_t num_cores = std::thread::hardware_concurrency();
```

Comment:
we have num_cores() define in the `util/cpu_info.h`

Reply:
that cpu_info has a bug, it always equal to 1